### ISSUE TITLE
Exception: Certificate "4131" has been revoked

### ISSUE THREAD

**Issue by Snake883** [pbek/ownbackup#42]
I am trying to download/install Ownbackup...

Error Message:

> Exception: Certificate "4131" has been revoked

> [settings] Error: Exception: Certificate "4131" has been revoked at <<closure>>
> 
> 0. /var/www/nextcloud/settings/Controller/AppSettingsController.php line 444
>    OC\Installer->downloadApp("ownbackup")
> 1. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 170
>    OC\Settings\Controller\AppSettingsController->enableApps(["ownbackup"], [])
> 2. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 99
>    OC\AppFramework\Http\Dispatcher->executeController(OC\Settings\Cont ... {}, "enableApps")
> 3. /var/www/nextcloud/lib/private/AppFramework/App.php line 126
>    OC\AppFramework\Http\Dispatcher->dispatch(OC\Settings\Cont ... {}, "enableApps")
> 4. /var/www/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php line 47
>    OC\AppFramework\App::main("OC\\Settings\\C ... r", "enableApps", OC\AppFramework\ ... {}, {_route: "settin ... "})
> 5. <<closure>>
>    OC\AppFramework\Routing\RouteActionHandler->__invoke({_route: "settin ... "})
> 6. /var/www/nextcloud/lib/private/Route/Router.php line 297
>    undefinedundefinedcall_user_func(OC\AppFramework\ ... {}, {_route: "settin ... "})
> 7. /var/www/nextcloud/lib/base.php line 997
>    OC\Route\Router->match("/settings/apps/enable")
> 8. /var/www/nextcloud/index.php line 42
>    OC::handleRequest()

>  POST /nextcloud/index.php/settings/apps/enable

Nextcloud v17.0.2

**Comment by pbek** [comment]
Yes, the app was renamed to Nextbackup, please install the new app.

See why: https://github.com/pbek/nextbackup/blob/develop/CHANGELOG.md#19120

**Comment by Snake883** [comment]
Should Nextcloud not list OwnBackup as something to install?
![image](https://i.imgur.com/LjoGhLt.png)

**Comment by pbek** [comment]
They still have to remove or disable it, see https://github.com/nextcloud/app-certificate-requests/pull/276
